clients/gtest/trsm_ex_gtest.cpp

```diff
@@ -7,13 +7,72 @@
 #include "rocblas_datatype2string.hpp"
 #include "rocblas_test.hpp"
 #include "testing_trsm_ex.hpp"
+#include "testing_trsm_ex_batched.hpp"
+#include "testing_trsm_ex_strided_batched.hpp"
 #include "type_dispatch.hpp"
 #include <cctype>
 #include <cstring>
 #include <type_traits>
 
 namespace
 {
+    // possible trsm_ex test cases
```

Comment:
Just noticed that gemm and gemm_ex are combined into a single *gtest.cpp file. The same could be done for trsm.

Reply:
Combined trsm_gtest.cpp and trsm_ex_gtest.cpp into a single trsm_gtest.cpp in 9d378b0.